case_mapping/uco/observable.py

```diff
@@ -702,23 +708,21 @@ def __init__(
                 "uco-observable:application": application,
                 "uco-observable:from": call_from,
                 "uco-observable:to": call_to,
+                "uco-observable:participant": call_participant,
```

Comment:
A call most often has multiple participants.  Unfortunately, this can't remain a single-valued property, it needs to be a Python list & JSON array.

Reply:
I add the uco-observable:participant property to example.py but the doubt about this property remains. I could have inserted the additional participant to the uco-observable:to property that can be represented by an array of ObjectObservable containing PhoneAccountFacet or ApplicationAcocuntFacet.

Reply:
By my recollection, the participants array was meant to be a bit redundant with the `from` and `to` properties.

A call could be between just people, and in that case the members of the `participant` list would be what's in `from` and `to`.  A call could also be between more than two people, in which case there might be a `from` (who started a call), there might be `to`'s (e.g., people initially invited to a call, as noted by the conference service), and there might be other `participant`s (those who joined a call via an invitation sent out-of-band of the conference service).

Informally, `from` and `to` are specializations of `participant`.  UCO hasn't formally encoded that relationship between the properties yet.  There may be some nontrivial technical issues to work through.